addons/mail/static/src/components/chatter_topbar/chatter_topbar.xml

```diff
@@ -34,42 +37,42 @@
                         </button>
                     </t>
                     <t t-if="chatter.hasActivities">
-                        <button class="btn btn-link o_ChatterTopbar_button o_ChatterTopbar_buttonScheduleActivity" type="button" t-att-disabled="chatter.isDisabled" t-on-click="chatter.onClickScheduleActivity" title="Schedule an activity" data-hotkey="shift+a">
+                        <button class="o_ChatterTopbar_button o_ChatterTopbar_buttonScheduleActivity btn btn-link rounded-0" type="button" t-att-disabled="chatter.isDisabled" t-on-click="chatter.onClickScheduleActivity" title="Schedule an activity" data-hotkey="shift+a">
                             <i class="fa fa-clock-o"/>
                             Schedule activity
                         </button>
                     </t>
                     <div class="o-autogrow"/>
-                        <div class="o_ChatterTopbar_rightSection">
-                            <button class="btn btn-link o_ChatterTopbar_button o_ChatterTopbar_buttonAttachments" type="button" t-att-disabled="chatter.isDisabled" t-on-click="chatter.onClickButtonAttachments">
+                        <div class="o_ChatterTopbar_rightSection flex-grow-1 flex-shrink-0 justify-content-end d-flex">
+                            <button class="o_ChatterTopbar_button o_ChatterTopbar_buttonAttachments btn btn-link rounded-0" type="button" t-att-disabled="chatter.isDisabled" t-on-click="chatter.onClickButtonAttachments">
                                 <i class="fa fa-paperclip"/>
                                 <t t-if="!chatter.isShowingAttachmentsLoading">
-                                    <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonAttachmentsCount" t-esc="chatter.thread ? chatter.thread.allAttachments.length : 0"/>
+                                    <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonAttachmentsCount pl-1" t-esc="chatter.thread ? chatter.thread.allAttachments.length : 0"/>
                                 </t>
                                 <t t-if="chatter.isShowingAttachmentsLoading">
-                                    <i class="o_ChatterTopbar_buttonAttachmentsCountLoader fa fa-circle-o-notch fa-spin" aria-label="Attachment counter loading..."/>
+                                    <i class="o_ChatterTopbar_buttonAttachmentsCountLoader fa fa-circle-o-notch fa-spin ms-1" aria-label="Attachment counter loading..."/>
                                 </t>
                             </button>
                             <t t-if="chatter.hasFollowers and chatter.thread">
                                 <t t-if="chatter.thread.channel_type !== 'chat'">
                                     <FollowButton
-                                        className="'o_ChatterTopbar_button o_ChatterTopbar_followButton'"
+                                        className="'o_ChatterTopbar_button o_ChatterTopbar_followButton rounded-0'"
```

Comment:
I know that the raw SCSS for these child-components  is currently defined here, but it's not the right place.

I think we should open parallel PRs and add these `d-flex rounded-0` classes in  `FollowButton` and `FollowerListMenu` direclty.

@seb-odoo ?

Reply:
I fail to see the relevance of `border-radius: 0;`/`rounded-0` here.
With or without this style, the item still have squared borders.
We could just remove `rounded-0`.

Reply:
Hello @stefanorigano @alexkuhn 👋 ,

>  I think we should open parallel PRs and add these d-flex rounded-0 classes in FollowButton and FollowerListMenu direclty.

You’re right, it’s better to put the classes in the component. I revamped the FollowerListMenu at the same time in another PR: https://github.com/odoo/odoo/pull/86727.
I also cherry-picked the commit from this PR to show the final render. Should I keep it like this or would you prefer that I remove it ?

FollowButton doesn’t seem to need the `rounded-0`.

> I fail to see the relevance of border-radius: 0;/rounded-0 here.

![border-radius](https://user-images.githubusercontent.com/80679690/159241472-7cdae667-40f9-4321-a3cb-7c44ddcd372f.jpg)

The `rounded-0` is applied for the Community which allows rounded borders and doesn’t look good when the `composer` is displayed.
To keep consistency with the Topbar buttons, I think it’s better to keep the `rounded-0`, what do you think ?

Reply:
I think that these buttons should be designed consistently in each version, meaning that should be rounded in community and squared  in enterprise.

I'd suggest:
`border-radius: border-radius: $nav-tabs-border-radius $nav-tabs-border-radius 0 0;`

Reply:
> > I fail to see the relevance of border-radius: 0;/rounded-0 here.
> 
> ![border-radius](https://user-images.githubusercontent.com/80679690/159241472-7cdae667-40f9-4321-a3cb-7c44ddcd372f.jpg)
> 
> The `rounded-0` is applied for the Community which allows rounded borders and doesn’t look good when the `composer` is displayed. To keep consistency with the Topbar buttons, I think it’s better to keep the `rounded-0`, what do you think ?

Ok, this is just community style.
I don't mind doing that in another PR. Better keep this PR simple and just turn SCSS to bootstrap classes.

Reply:
I've fixed this design inconsistency in another PR : https://github.com/odoo/odoo/pull/87148

cc. @stefanorigano